src/platform_impl/macos/monitor.rs

```diff
@@ -22,17 +22,44 @@ use core_video_sys::{
     CVDisplayLinkGetNominalOutputVideoRefreshPeriod, CVDisplayLinkRelease,
 };
 
-#[derive(Derivative)]
-#[derivative(Debug, Clone, PartialEq, Hash)]
+#[derive(Clone, Eq)]
```

Comment:
Technically `Eq` wasn't here before, but it looks like it should have been so I added it.